### ISSUE TITLE
Support AWS CodePipelines

### ISSUE THREAD

**Issue by jakepage91** [tailwarden/komiser#1176]
**Describe the solution you'd like**
Collect AWS CodePipelines
Additional context
To get an idea of how to implement the collector, take the [EC2 instances](https://github.com/tailwarden/komiser/blob/master/providers/aws/ec2/instances.go) resource as inspiration.

> Note: only support the AWS resource, please don't add any cost calculation logic

**Comment by jakepage91** [comment]
Hey there @subhajit20. This could be a great one for you to tackle. Here is a video that helps you get started https://www.youtube.com/watch?v=Vn5uc2elcVg

**Comment by jakepage91** [comment]
All yours!

**Comment by bishal7679** [comment]
> hey @jakepage91 , I have seen the tutorial you mentioned... have set up the project locally but in this case, how do I know which method to use for implementing codePipelines? There are multiple methods available in the documentation

You can take a look for ec2 instance.go file for which function to use and as we are currently focusing for service level cost rather resource level so it'd be a good start to only fetch the service from regions

**Comment by jakepage91** [comment]
Yeah I haven't gone through the available methods for AWS Code Pipelines but the idea is to simply list each pipeline resource (forget about cost at the moment) and make sure it shows up in the resource inventory

**Comment by subhajit20** [comment]
Okay,
@jakepage91  one more thing is I have added the `config.toml` file to the project but do not understand how to generate the credential file ?
this i s why the cloud account could not be found.
Can you help out ?
![3000-subhajit20-komiser-ob5wnr61k30 ws-us106 gitpod io_dashboard_ - Google Chrome 11-11-2023 13_17_55](https://github.com/tailwarden/komiser/assets/99127578/7f312d86-d4ee-41d5-9c6d-8f0cc6102dd9)

**Comment by bishal7679** [comment]
> Okay, @jakepage91 one more thing is I have added the `config.toml` file to the project but do not understand how to generate the credential file ? this i s why the cloud account could not be found. Can you help out ? ![3000-subhajit20-komiser-ob5wnr61k30 ws-us106 gitpod io_dashboard_ - Google Chrome 11-11-2023 13_17_55](https://user-images.githubusercontent.com/99127578/282228583-7f312d86-d4ee-41d5-9c6d-8f0cc6102dd9.png)

Hey @subhajit20 You can set your credentials by AWS Cli 
`aws configure` command
Komiser will automatically fetch it when you mention `source="ENVIRONMENT_VARIABLES"` in config.toml
